spec/errors.md

```diff
@@ -269,7 +269,7 @@ a reference to a function which cannot be resolved.
 ### Bad Selector
 
 A **_<dfn>Bad Selector</dfn>_** error occurs when a message includes a _selector_
-with a resolved value which does not support selection.
+with a _resolved value_ which does not support selection.
```

Comment:
I'm confused. Can you give an example of a selector that supports selection where its resolved value doesn't?

That is, if I have
.input {$num :number}
.match $num
...
How could :number ever have a resolved value that didn't support selection? I think the notion of 'resolved value' doesn't help.

```suggestion
that does not support selection.
```

Reply:
> How could :number ever have a resolved value that didn't support selection?

Going with the example you posted above, if that message is formatted without a `num` argument, then the first step of [Function Resolution](https://github.com/unicode-org/message-format-wg/blob/main/spec/formatting.md#function-resolution) applies:

> If the _expression_ includes an _operand_, resolve its value. If this fails, use a _fallback value_ for the _expression_.

And then this part of [Fallback Resolution](https://github.com/unicode-org/message-format-wg/blob/main/spec/formatting.md#fallback-resolution) applies:

> _Pattern selection_ is not supported for _fallback values_.

And so we end up emitting two errors when formatting:
1. Unresolved Variable
2. Bad Selector

Reply:
I think @macchiati's finger is on the right thing. It's not a technical problem. This is just hard to understand. Perhaps:

```suggestion
A **_<dfn>Bad Selector</dfn>_** error is an error that occurs when a _message_
includes a _selector_ whose _resolved value_ 
contains a _function handler_ which does not support selection,
is missing a _function handler_,
or consists of a _fallback value_.
```

Reply:
Could we spin this suggestion off into a separate PR? As is, the only change that is being applied here is linkifying _resolved value_.

Reply:
Works for me as a solution.